sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GoneAndRetryWithRetryPolicy.java

```diff
@@ -60,13 +60,27 @@ public GoneAndRetryWithRetryPolicy(RxDocumentServiceRequest request, Integer wai
             !(exception instanceof RetryWithException) &&
             !(exception instanceof PartitionIsMigratingException) &&
             !(exception instanceof InvalidPartitionException &&
-            (this.request.getPartitionKeyRangeIdentity() == null ||
-            this.request.getPartitionKeyRangeIdentity().getCollectionRid() == null)) &&
+                (this.request.getPartitionKeyRangeIdentity() == null ||
```

Comment:
NIT: the the indent same as above line?

Reply:
No - intentional - because it is the second condition for the logical AND.

Reply:
oh, yea, sorry, miss read )